src/js/tech/html5.js

```diff
@@ -70,6 +72,13 @@ class Html5 extends Tech {
       }
     }
 
+    if (this.featuresNativeVideoTracks) {
+      this.handleVideoTrackChange_ = Fn.bind(this, this.handleVideoTrackChange);
```

Comment:
I'm still confused :) Why weren't they bound properly? By "vendor", do you mean the HTML tech or Flash tech for example?